File: verify_update.js
const { createClient } = require('@supabase/supabase-js');
const dotenv = require('dotenv');
const path = require('path');

dotenv.config({ path: path.resolve(__dirname, 'backend/.env') });

const url = process.env.SUPABASE_URL;
const key = process.env.SUPABASE_SERVICE_ROLE_KEY;
const supabase = createClient(url, key);

async function verify() {
    const { data, error } = await supabase
        .from('inventory_items')
        .select('*')
        .in('code', ['BB2301', 'CLP0801'])
        .order('code');

    if (error) {
        console.error('Error:', error);
        return;
    }

    console.log('Verification Data:');
    console.log(JSON.stringify(data, null, 2));
}

verify();

File: verify_update.py
import pandas as pd
import requests
import os
from dotenv import load_dotenv
import json

load_dotenv('backend/.env')

SUPABASE_URL = os.getenv('SUPABASE_URL')
SUPABASE_SERVICE_ROLE_KEY = os.getenv('SUPABASE_SERVICE_ROLE_KEY')

HEADERS = {
    "apikey": SUPABASE_SERVICE_ROLE_KEY,
    "Authorization": f"Bearer {SUPABASE_SERVICE_ROLE_KEY}"
}

def verify():
    codes = ['BB2301', 'CLP0801']
    url = f"{SUPABASE_URL}/rest/v1/inventory_items?code=in.({','.join(codes)})"
    response = requests.get(url, headers=HEADERS)
    if response.status_code == 200:
        print("Verification Data:")
        print(json.dumps(response.json(), indent=2))
    else:
        print(f"Error: {response.status_code} - {response.text}")

if __name__ == '__main__':
    verify()
